src/app/components/acf/acf.html

```diff
@@ -24,13 +24,15 @@ <h2 class="panel-title" ng-if="vm.mode === 'edit'">Edit current ACF</h2>
           <div class="form-group" ng-class="{ 'has-error' : (vm.showFormErrors || vm.queryForm.selectAcf.$touched) && vm.queryForm.selectAcf.$error.required }" ng-if="!vm.acfWritesAllowed">
             <div class="col-sm-6">
               <label for="selectAcfPrefix" class="control-label sr-only">ACF County Selection</label>
+              <label for="selectAcfPrefix" class="control-label">ACF County Selection</label>
```

Comment:
In this case the line above the marked one already has a label for the form element. It's required to satisfy 508 requirements. However, I tend to hide the element from view when I think it's "obvious" what the element is for. The class `sr-only` is a bootstrap helper class that says "This element should be moved off the screen visually, but left in the page for screenreaders to parse". 

I can see two separate ways of solving the problem. The first is to remove the `sr-only` class, and that will just let the label show. I don't really like that, because I don't know if it's really necessary. The other option is to make the first option in each select box something that tells the user what to do without being an actual label. Something like `<option disabled="true" selected="true">Choose an ACF County</option>`.

Oh, and having two labels pointing at the same form element is actually anti-508.

Same comments for the below box.